src/App.js

```diff
@@ -1,56 +1,56 @@
-import React, { useState } from "react"
-import moment from "moment"
-import Confetti from "react-confetti"
-import { Spin } from "antd"
-import { width, height } from "./utils/Dimensions"
-import { usePrevious } from "./utils/usePrevious"
-import { FaceBookIcon, TwitterIcon } from "./assets/svgs"
-import HomeShowcase from "./components/HomeShocase"
-import SearchForm from "./components/SearchForm"
-import SocialHandler from "./components/SocialHandler"
-import ErrorMessage from "./components/ErrorMessage"
-import Profile from "./components/Profile"
-import GithubService from "./services/Github"
-import "./App.css"
+import React, { useState } from "react";
+import moment from "moment";
+import Confetti from "react-confetti";
+import { Spin } from "antd";
+import { width, height } from "./utils/Dimensions";
+import { usePrevious } from "./utils/usePrevious";
+import { FaceBookIcon, TwitterIcon, LinkedinIcon } from "./assets/svgs";
+import HomeShowcase from "./components/HomeShocase";
+import SearchForm from "./components/SearchForm";
+import SocialHandler from "./components/SocialHandler";
+import ErrorMessage from "./components/ErrorMessage";
+import Profile from "./components/Profile";
+import GithubService from "./services/Github";
+import "./App.css";
```

Comment:
remove semi-colon from changed files because we are using the no-semi-colon approach

Reply:
Sure i can remove them and Actually this happened because of prettier extension ..